core/storage/collection/gae_models.py

```diff
@@ -162,6 +162,8 @@ class CollectionRightsModel(base_models.VersionedModel):
             constants.ACTIVITY_STATUS_PUBLIC
         ]
     )
+    # DEPRECATED in v2.8.1. Do not use.
```

Comment:
2.8.1 seems wrong. What release is this being targeted for?

Reply:
Changed it to 2.8.2.